cloudinit/sources/DataSourceOVF.py

```diff
@@ -358,8 +358,11 @@ def _get_data(self):
                 if contents:
                     break
             if contents:
-                (md, ud, cfg) = read_ovf_environment(contents)
+                read_network = ('com.vmware.guestinfo' == name)
```

Comment:
Should this not have been `com.vmware.guestInfo` (different capitalization) the same as in line 353 above?

Reply:
Also why is this not supposed to work with the ISO transport?

Reply:
@t-8ch Thanks for catching this. I should add more tests on this change. Will file a PR to fix it and add tests.

Reply:
@t-8ch For ISO transport, It can be supported, but user need create ISO for each instance with specified network configuration, I don't see it's a common user case when I opened this PR.
And  `com.vmware.guestInfo` is VMware platform specified, I get this request from inside of VMware, the guestinfo.ovfEnv can be updated easily from outside of instance(vm).

Reply:
Would not VMware itself be able to generate the ISO on demand, at least when specifying the ISO transport in VMware?

Reply:
Yes, AFAIK, ISO can be generated on demand on VMware platform, till now, network config is never included in such ISO.  The ISO transport is not VMware specified, if there is indeed such requirement, I can make the change and let cloud-init team review it.

Reply:
@t-8ch Please fell free to send the patch, I was hesitating to add ISO transport because it's not specified to VMware platform. Let's see if cloud-init team is ok with it too.